utils/led.h

```diff
@@ -11,17 +11,14 @@
 
```

Comment:
Can you please add the GPL header to the top of the file (copy over from other files)?

Reply:
yes, done